static/scss/product-details.scss

```diff
@@ -11,6 +11,13 @@
     margin: 0 auto;
   }
 
+  .btn-primary.highlight {
```

Comment:
should this not come in the `common.scss` file in `.btn-primary` class?

Reply:
It's true that only the product detail page has this button with the `highlight` class, but the spacing of this button is specific to the product detail page, and we may have other pages in the future where we want a `highlight` button but we don't want the same spacing.